src/components/FeatureFlags/unleashClient.ts

```diff
@@ -0,0 +1,27 @@
+import { UnleashClient } from '@unleash/proxy-client-react';
+
+let unleashClient: UnleashClient;
```

Comment:
FWIW some of this came from the provider here: https://github.com/RedHatInsights/insights-chrome/pull/2835/files#diff-4a88bdbf8236baa86cca130fd1db3eecce2e43da7bf613914792361d031b728bL54-L62

👍 for separating it out